Homework_12/contact-us.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact us</title>
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="stylesheet" href="styles/header.css" />
+    <link rel="stylesheet" href="styles/general.css" />
+    <link rel="stylesheet" href="styles/contact-us.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <img src="img/peugeot-logo-2021-1000x1100-1-273x300 1 (1).png" />
+    </header>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="#">Gallery</a>
+      <a href="#">Contact us</a>
+    </nav>
+    <main>
+        <h1>Contact us</h1>
+      <form class="contact-form" autocomplete="off" method="post" action="somefile.php" name="contact-form">
+        <label for="first-name-input">First Name</label>
+        <input type="text" name="first-name-input" id="first-name-input" />
+
+        <label for="last-name-input">Last Name</label>
+        <input type="text" name="last-name-input" id="last-name-input" />
+
+        <label for="email-input">Email</label>
+        <input type="email" name="email-input" id="email-input" />
+
+        <label for="phone-input">Phone number</label>
+        <input type="tel" name="phone-input" id="phone-input"/>
+        <div class="submit-container">
+        <input type="submit" value="Send" class="submit">
+        </div>
+  </form>
+      </form>
+      <footer>
```

Comment:
you know your mistake:)

next time not be shy t fix it by another commit
you can edit your Pull Request as many times as you want by pushing new changes